### ISSUE TITLE
Set up AWS frontend resources 

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#3798]
Set up Cloudfront distributions and domains (Route 53, city, etc.) and linking them accordingly.

**Comment by sergiogcx** [comment]
@mike set up the Netlify resources, I've shown him around AWS CloudFront and we set up the Distributions for both. The domain was set up for staging, and once the CloudFront distribution was set up I requested the DNS change for mobility.austin.gov, the ticket number is `SCTASK0094001`.

During the day I had been talking back and forth with Lucky, there was some concern the domain did not support CNAMEs, but after some testing it was determined that it is in fact supported.

The next aspect is in relation to the domain being not only a DNS migration but a service migration, from the communications it reads:

```
So, using austintx.gov I verified that we can use a CNAME for the subdomain origin. In order to make this change to austin.gov somebody from your team will need to put in a change management request since it affects production workflows. If you have questions about the CMB process, please let me know. As far as scheduling, we prefer to make these changes earlier in the week.
Even though it's just a DNS change, it's actually a service migration.  There also needs to be a knowledge base article letting the service desk know that issues for mobility.austin.gov webpages need to go to ATD and not the web team or eig.
Please let me know if you have any questions.
```

Replied, waiting on their confirmation.

**Comment by sergiogcx** [comment]
I noticed we need to configure the react router to work with /moped, this may be high priority. Will get to this as soon as I finish another issue.

**Comment by sergiogcx** [comment]
Closing, this has been done.
